docs/schema.md

```diff
@@ -114,3 +117,15 @@ For this URL: <https://www.algolia.com/products/crawler/>
 ## Record splitting
 
 For better relevance, we can split records into multiple ones. We create all indices with the index settings `{ distinct: true, attributeForDistinct: 'url' }` to deduplicate them at search time.
+
+## JSON-LD
+
+We support a limited set of [JSON-LD](https://json-ld.org/) attributes. We expect the JSON-LD structure to follow the https://schema.org/ structure.
```

Comment:
```suggestion
We support a limited set of [JSON-LD](https://json-ld.org/) attributes. We expect the JSON-LD structure to follow the <https://schema.org/> structure.
```

Reply:
Added, but it looks the same with or without those.

Reply:
it's because github is kind to autoparse url probably, but it's more correct like this